serve.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh -ex
+
+cd /app
+npm start -- -l tcp://0.0.0.0:${PORT-8001}
```

Comment:
We should use 8004 as a default port here. We then have default ports of 8001 for ionic, 8002 for nest, 8003 for poster, and 8004 for metadata.

```suggestion
npm start -- -l tcp://0.0.0.0:${PORT-8004}
```